doc TokenLanguageModelKNN: Add centroid_column option (#2665)
groonga/groonga: doc/source/reference/tokenizers/token_language_model_knn.md

`query_prefix` specifies the prefix for query text.

(tokenizer-language-model-knn-centroid-column)=

#### `centroid_column`

```{versionadded} 15.2.1

```

This option is for large embeddings with more than 1025 dimensions (more than 4100 bytes).
Groonga table keys must be 4 KiB or smaller. Embeddings larger than 4 KiB cannot be stored as keys.

> You can use this option to store large embeddings in column values instead of table keys.

Reviewer: [nits]
```suggestion
You can use this option to store large embeddings in a column instead of in the table key.
```
